src/components/chart.svelte

```diff
@@ -91,6 +93,38 @@
         }
     }
 
+    function calculateMaxYValue(maxY, steps) {
+        if (maxY < 2) return Math.ceil(maxY)
+        const roundedToTen = Math.ceil(maxY / 10) * 10
+        return Math.ceil(roundedToTen / steps) * steps
+    }
+
+    function calculateMinYValue(minY, steps) {
+        if (minY < 2) return Math.floor(minY)
+        const roundedToTen = Math.floor(minY / 10) * 10
+        return Math.floor(roundedToTen / steps) * steps
+    }
+
+    function drawYLabels(min, max, steps) {
+        let maxYValue = calculateMaxYValue(max, steps)
+        let minYValue = calculateMinYValue(min, steps)
+        let range = maxYValue - minYValue
+        let yFrequency = range / (steps - 1)
+        let ySpacing = (chartHeight - 7) / (steps - 1)
```

Comment:
const Y_AXIS_SPACING = 7 at the top of the file with other constants 
const segments = steps - 1 at the top of this function?

^ at the top